src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -127,7 +127,7 @@ export default function SavedObjectFactory(esAdmin, kbnIndex, Promise, Private,
      * @return {Promise}
      * @resolved {SavedObject}
      */
-    this.init = _.once(() => {
+    this.init = _.once((isDefered) => {
```

Comment:
A few observations/questions:
 - `_.once()`'d functions should almost never accept arguments
 - What does `isDefered` mean? What is/should be defered? The Saved Object?
 - Calling the `docSource._createRequest` method from here feels like a violation of the intended public api for the docSource. Why not just change how `docSource.fetch()` functions?